mslib/mswms/_tests/test_wms.py

```diff
@@ -367,11 +371,46 @@ def test_multiple_xml(self):
         callback_ok_xml(result.status, result.headers)
 
     def test_import_error(self):
-        import mslib.mswms.wms
         with mock.patch.dict("sys.modules", {"mss_wms_settings": None, "mss_wms_auth": None}):
             reload(mslib.mswms.wms)
             assert mslib.mswms.wms.mss_wms_settings.__file__ is None
             assert mslib.mswms.wms.mss_wms_auth.__file__ is None
         reload(mslib.mswms.wms)
         assert mslib.mswms.wms.mss_wms_settings.__file__ is not None
         assert mslib.mswms.wms.mss_wms_auth.__file__ is not None
+
+    def test_gallery(self):
+        tempdir = tempfile.mkdtemp()
```

Comment:
Are this directories removed after the test? Perhaps use tmpdir fixture and two sub-directories. pytest then takes care of removing the tmpdir.

Reply:
Done